src/d3d12/d3d12_textures.cpp

```diff
@@ -725,4 +725,126 @@ namespace wr::d3d12
 		return out_format;
 	}
 
+	std::size_t SizeOfFormat(Format format)
```

Comment:
I remember adding this to some utility file a few months ago. Is this duplicate code, or was the old function removed?